src/components/inspectors/CycleExpression.vue

```diff
@@ -68,10 +83,25 @@ export default {
       if (this.periodicity.isTime) {
         return `R/PT${this.repeat}${this.periodicity.value}`;
       }
+      if (this.periodicity.isWeek && this.expression) {
+        return this.expression;
+      }
       return `R/P${this.repeat}${this.periodicity.value}`;
```

Comment:
The `cycleExpression` computed property only appears to be used for the watcher, and PHPStorm complains about this function being unused. But I think keeping the computed property makes this easier to read.